packages/gatsby-cli/src/reporter/loggers/ink/cli.tsx

```diff
@@ -2,23 +2,50 @@ import React from "react"
 import { Box, Static } from "ink"
 import { isTTY } from "../../../util/is-tty"
 import { trackBuildError } from "gatsby-telemetry"
-
-import { Spinner } from "../ink/components/spinner"
-import { ProgressBar } from "../ink/components/progress-bar"
-import { Message } from "../ink/components/messages"
-import { Error } from "./components/error"
-import Develop from "../ink/components/develop"
+import { Spinner } from "./components/spinner"
+import {
+  ProgressBar,
+  IProps as IProgressbarProps,
+} from "./components/progress-bar"
+import { Message, IProps as IMessageProps } from "./components/messages"
```

Comment:
I think it would be better if we just stopped naming everything as `IProps` and give them better names all over so we don't have to do these renames

Reply:
No problemo. Copied this naming from #22116 actually